src/utils.js

```diff
@@ -8,25 +9,12 @@ function delay(ms) {
   });
 }
 
-export { getRandomNumber, delay };
+function copyObject(obj) {
+  return { ...obj };
+}
+
+function getQuotient(parent, child) {
+  return Math.floor(parent, child);
```

Comment:
```suggestion
  return Math.floor(child / parent);
```

- 몫을 구하는 유틸 함수인데 이런 형태를 의도하신걸까요?
- 사용처가 없는데, `ModifiableInput` 컴포넌트에서 사용할 수 있을 거 같네요.